pandas/core/series.py

```diff
@@ -5993,6 +5993,58 @@ def mask(
 
     index = properties.AxisProperty(
         axis=0, doc="The index (axis labels) of the Series."
+            """
```

Comment:
Can you make this a single text, not concatenating in this way please?

Reply:
Sorry, but didn't understand. You mean like this ?
axis=0, doc="The index (axis labels) of the Series."  """  A one-dimensional ndarray with hashable type axis labels. """ 

or this 

axis=0, doc="The index (axis labels) of the Series."  
"""  A one-dimensional ndarray with hashable type axis labels. """